packages/language-chooser/src/index.ts

```diff
@@ -0,0 +1 @@
+export { default as LanguageChooser } from './language-chooser';
```

Comment:
Should we export `LanguageChooser` as a private API at least initially, to give us time to test it and make it public once we feel good about it?

Reply:
Isn't this what semver is for?

Can't really make this a private API as this will need to be used in WP core itself, not in Gutenberg. Not really a place where we can or should use `@wordpress/private-apis`.

Reply:
Since we can't use `@wordpress/private-apis` in core, what did you invision?

Reply:
I didn't initially consider that `@wordpress/private-apis` couldn't be used.

In case we wanted to work iteratively with follow-up PRs until we feel confident in exposing the component as a public API, an alternative could be not to publish any APIs for now, and manually enable it any time we need to test it WPCore. Not great, but it would do the trick in the meantime.

Reply:
In that case we could just not publish the package to npm, no?

Reply:
Setting `private` to `true` also helps with that: https://docs.npmjs.com/cli/v10/configuring-npm/package-json#private